ci/jenkins/test-vmc.sh

```diff
@@ -309,14 +309,24 @@ function deliver_antrea {
     # be leveraged successfully
     chmod -R g-w build/images/ovs
     chmod -R g-w build/images/base
-    for i in `seq 2`
-    do
+    # Sometimes pulling images from Harbor results in: error pulling image configuration: unknown blob.
+    # Then try to pull images from dockerhub directly once.
+    BUILD_ANTREA_UBUNTU_SUCCESS=false
+    for i in `seq 2`; do
         if [[ "$COVERAGE" == true ]]; then
             VERSION="$CLUSTER" DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-ubuntu-all.sh --pull --coverage && break
         else
             VERSION="$CLUSTER" DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-ubuntu-all.sh --pull && break
         fi
+        BUILD_ANTREA_UBUNTU_SUCCESS=true
```

Comment:
This won't be executed when building succeeds. 
The change would make the script build the image twice when it succeeds and wouldn't retry with dockerhub when it fails.

I would defer to @antoninbas to see whether it makes sense to retry dockerhub, my intention was to improve the error reporting, it should exit with error immediately when building antrea images fails, instead of continuing the process and met an error in other place. I created https://github.com/antrea-io/antrea/pull/2184 to improve the error processing.

Reply:
@lzhecheng maybe we can start with Quan's PR and see if we need to try to pull from dockerhub in case of harbor failures. There have been quite a lot of issues with the harbor registry since they enabled caching with a CDN.

Reply:
Thanks @tnqn 's suggestion and I rebased my PR with #2184.